cmd/install/assets/hypershift_operator.go

```diff
@@ -494,6 +507,9 @@ func (o HyperShiftOperatorService) Build() *corev1.Service {
 			Labels: map[string]string{
 				"name": "operator",
 			},
+			Annotations: map[string]string{
```

Comment:
note that depending on this for plain vanilla kube based clusters would not be accurate (someone would need to add the cert. Not necessarily a problem just wanted to point it out.